OHFJIRA-70 : old gui cleanup

kkovarik opened the pull request:
Old GUI in web-admin module removal

#### CHANGES
* removed old gui resources (css, ftl templates etc...), controllers and configuration
* configured spring-security to work with new admin-console properly: removed all 3xx redirects, if not logged, unauthorized 401 is returned, if login fails then 403
* changed configuration of basic authentication for webservices, as it has not correctly worked in standalone mode (see comment in WebSecurityConfig)
* switched location of spring-boot maven plugin, -exec repackaged archive is now only in web-admin submodule

#### RELATED JIRA
* https://openhubframework.atlassian.net/browse/OHFJIRA-70

hanusto committed:
Improvement how changes controller can be configured - load name/location of changes.MD from property.

Issue: NOJIRA

kkovarik committed:
OHFJIRA-70 : cleanup of old gui resources (templates, styles, images etc.)

pjuza commented on web-admin/src/main/java/org/openhubframework/openhub/web/common/WebProps.java:
```diff
@@ -29,6 +29,8 @@
 
     public static final String MESSAGES_LIMIT = OpenHubPropertyConstants.PREFIX + "admin.console.messages.limit";
 
+    public static final String SESSION_CONCURRENCY_LIMIT = OpenHubPropertyConstants.PREFIX + "admin.console.session-concurrency.limit";
```
I added on wiki (https://openhubframework.atlassian.net/wiki/spaces/OHF/pages/33599/OpenHub+configuration)

pjuza reviewed (approved):
Big step to finish 2.x version. Thanks.

hanusto commented:
The fastest way how to increase code coverage - remove unused code lines :)

hanusto commented on web-admin/src/main/java/org/openhubframework/openhub/admin/web/log/LogController.java:
```diff
@@ -1,138 +0,0 @@
-import org.springframework.web.util.UriUtils;
-
-import org.openhubframework.openhub.admin.services.log.LogEvent;
-import org.openhubframework.openhub.admin.services.log.LogParser;
```
Please remove LogParser class, or maybe whole package _org.openhubframework.openhub.admin.services.log_

hanusto replied:
And message report service

kkovarik replied:
Removed, entire package & messageReportService as well.

hanusto commented on web-admin/src/main/java/org/openhubframework/openhub/admin/config/AdminSecurityConfig.java:
```diff
@@ -63,20 +84,18 @@
             http.csrf().disable() // HTTP with disabled CSRF
                     .authorizeRequests() //Authorize Request Configuration
-                        .antMatchers(WEB_URI_PREFIX + "homepage/**")
+                        .antMatchers(WEB_URI_PREFIX + "console/**")
```
Maybe we can reuse `spring.mvc.static-path-pattern` property.

kkovarik replied:
Don´t know, the WEB_URI_PREFIX has trailing slash /, and property now starts with slash as well... /console/**...

hanusto reviewed (changes requested):
Please try to remove log packages and unused classes.

kkovarik committed:
OHFJIRA-70 : cleanup of old gui controllers & configuration

- freemarker & jsp dependencies and configuration removed
- controllers and services needed only for the old gui removed

kkovarik committed:
OHFJIRA-70 : configuration of security to correctly work with admin-console

kkovarik committed:
NOJIRA : changed WS basic authentication configuration, to work correctly in standalone deployment
